gef-ui/server/utils/user-authorisation-level.js

```diff
@@ -0,0 +1,62 @@
+import { PROGRESS } from '../../constants';
+
+const authorisationLevel = {
+  READ: 'READ',
+  COMMENT: 'COMMENT',
+  EDIT: 'EDIT',
+};
+
+const statesWhereMakerHasEditAccess = [PROGRESS.DRAFT, PROGRESS.NOT_STARTED, PROGRESS.IN_PROGRESS];
+const statesWhereMakerHasReadAccess = [PROGRESS.BANK_CHECK];
+const statesWhereMakerHasCommentAccess = [PROGRESS.BANK_CHECK];
+const statesWhereCheckerHasEditAccess = [];
+const statesWhereCheckerHasReadAccess = [PROGRESS.BANK_CHECK];
```

Comment:
What should the CHECKERS access be when its in Draft, not_started or in_progress? Does this mean they don't have any access to the application?

Reply:
I beleive they dont see it in those states